### DISCUSSION TITLE
Github Codespaces Not Working On Firefox (Occasionally)

### DISCUSSION THREAD

**Discussion by Didward** [community/community#38956] [Codespaces] [answered]
### Select Topic Area

Bug

### CodeSpaces

Hey there, I hope you're having a great day! 👋

I am glad that GitHub has released free access to Codespaces! I created a Codespace (on Firefox) when I received an error (I don't have a screenshot of it) telling me that it was not working and that restarting/relaunching the Codespace could fix it and unfortunately it didn't.
I tried this consecutively and made no progress. 😕

I then went on to switch to Chrome and everything worked perfectly. After an hour or so, I decided to try it on Firefox and it worked. Annoyingly, this occurs a few times. Was this just a problem with my browser or was this a bug on Codespaces? 🤔

Thank you for reading! ☺️

**Comment by jfullstackdev** [comment, marked as answer]
probably on the side of the browser,

unfortunately, it may sound absurd for others, most developers prefer to adhere in Chrome than in FireFox,

also, maybe it's an issue of RAM management by Firefox, because even if Chrome is hogging the RAM of your computer, it's using it efficiently

**Reply by Didward** [reply]
I am shocked after reading that! Thanks for the response! 👍

**Reply by mjbear** [reply]
@Didward 
This may not be the actual answer (at least it wasn't in my case).

In my case it was a [browser privacy setting](https://github.com/orgs/community/discussions/124964#discussioncomment-9802403) and not memory management or compatibility.

**Comment by mjbear** [comment]
These threads might help others:
#124964 - [specifically this comment](https://github.com/orgs/community/discussions/124964#discussioncomment-9802403)
#42091
